components/EliminatedStatus/EliminatedStatus.tsx

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+/**
+ * @returns EliminatedStatus component
+ */
+export const EliminatedStatus = (): React.JSX.Element => {
+  return <p className="uppercase">eliminated</p>;
```

Comment:
@CorinaMurg Is this ideal to use CSS to uppercase the status? Should this be a heading or have a role?

Reply:
I'm curious about this as well, so thank you for asking Corina.

Reply:
@shashi @ryandotfurrer 
Yes, this should be wrapped in a container with role `alert` or `status` or with `aria-live` attribute, otherwise the change won't be announce to screen readers.
I added the link to an article on this topic in the discord thread for this PR

Reply:
Two things:
1. If the update might happen as the user is on the page, then we need a role role status/alert or an aria-live attribute. I added an article on this in the discord thread for this PR.
2. Regardless of the point above, I suggest using `aria-describedby` for each entry"
 `aria-describedby = id of the alive/eliminated`
 So for a screen reader user, when the entry name is announced, it's followed by "alive" or "eliminated" because it's crucial info the user needs to know re: that entry. I'll confirm with Homer just to be sure.

Reply:
@shashilo @jasonetorres How do you envision this working and what solution that Corina presented above would you like me to implement?

Reply:
@jasonetorres Let's a create a ticket to track this as a separate task.